lib/util/include/hse_util/hse_err.h

```diff
@@ -77,24 +77,23 @@ extern char hse_merr_bug2[];
  *   ------  -----  ----------
  *   63..48   16    signed offset of (_he_merr_file - merr_base) / MERR_ALIGN
  *   47..32   16    line number
- *   31..31    1    reserved bits
- *   30..0    31    positive errno value
+ *   31..16   16    error context
+ *   15..0    16    positive errno value
  */
 #define MERR_FILE_SHIFT (48)
 #define MERR_LINE_SHIFT (32)
-#define MERR_RSVD_SHIFT (31)
+#define MERR_ECTX_SHIFT (16)
 
 #define MERR_FILE_MASK  (0xffff000000000000ul)
 #define MERR_LINE_MASK  (0x0000ffff00000000ul)
-#define MERR_RSVD_MASK  (0x0000000080000000ul)
-#define MERR_ERRNO_MASK (0x000000007ffffffful)
+#define MERR_ECTX_MASK  (0x00000000ffff0000ul)
+#define MERR_ERRNO_MASK (0x000000000000fffful)
```

Comment:
I would just name them MERR_CTX_SHIFT and MERR_CTX_MAX, error is already in the name..  Then it lines up nicely with merr_errno(), merr_file(), merr_line(), and merr_ctx() as the merr getters...